components/service/fretboard/src/main/java/mozilla/components/service/fretboard/scheduler/workmanager/WorkManagerSyncScheduler.kt

```diff
@@ -35,7 +35,7 @@ class WorkManagerSyncScheduler {
         val constraints = Constraints.Builder()
             .setRequiredNetworkType(NetworkType.CONNECTED)
             .build()
-        val syncWork = PeriodicWorkRequest.Builder(SyncWorker::class.java, 1, TimeUnit.DAYS)
+        val syncWork = PeriodicWorkRequest.Builder(worker, 1, TimeUnit.DAYS)
             .setConstraints(constraints)
             .build()
         schedule(syncWork)
```

Comment:
nit: It looks like `schedule(request: WorkRequest)` doesn't need to be public. We may even be able to call `WorkManager.getInstance().enqueue()` here directly?

Reply:
Changed to private :)